src/lib/GraphDataUtils.js

```
@@ -0,0 +1,574 @@
+      if (!columns[xColumn] || !columns[yColumn]) {
+        return { isValid: false, error: 'Selected columns do not exist' };
+      }
+      break;
+
+    case 'pie':
+    case 'doughnut':
+      if (!valueColumn || (Array.isArray(valueColumn) && valueColumn.length === 0)) {
+        return { isValid: false, error: 'Pie charts require at least one value column' };
+      }
+      // Validate all value columns exist
+      const pieValueCols = Array.isArray(valueColumn) ? valueColumn : [valueColumn];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)